server/models/User.js

```diff
@@ -55,7 +55,7 @@ User.prototype.validPassword = function (password) {
 }
 
 sequelize.sync()
-	.then(() => console.log('users table has been successfully created, if one doesn\'t exist'))
-	.catch(error => console.log('This error occured', error));
+// .then(() => console.log('users table has been successfully created, if one doesn\'t exist'))
+// .catch(error => console.log('This error occured', error));
```

Comment:
Ideally you should delete lines rather than comment them out.

Reply:
Ah yes, thanks for spotting this